src/transformState.js

```diff
@@ -5,7 +5,31 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (true) {
+      case action.type === 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case action.type === 'removeProperties':
+        removeProperties(state, action.keysToRemove);
+        break;
+
+      case action.type === 'clear':
+        removeProperties(state, Object.keys(state));
+        break;
+    }
```

Comment:
```suggestion
    switch (action.type) {
      case 'addProperties':
        Object.assign(state, action.extraData);
        break;

      case action.type === 'removeProperties':
        removeProperties(state, action.keysToRemove);
        break;

      case action.type === 'clear':
        removeProperties(state, Object.keys(state));
        break;
    }
```
check appropriate usage of switch and fix all cases as I show